nmigen_boards/rz_easyfpga_a2_2.py

```diff
@@ -0,0 +1,129 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.intel import *
+from .resources import *
+
+__all__ = ["RZEasyFPGAA2_2Platform"]
+
+class RZEasyFPGAA2_2Platform(IntelPlatform):
+    device      = "EP4CE6" # Cyclone IV 6K LEs
+    package     = "E22"    # EQFP 144 pins
+    speed       = "C8"
+    default_clk = "clk50" # 50MHz builtin clock
+    resources   = [
+        # Clock
+        Resource("clk50", 0, Pins("23", dir="i"),
+                 Clock(50e6), Attrs(io_standard="3.3-V LVTTL")),
+
+        # LEDs, located on the bottom of the board.
+        *LEDResources(
+            pins="87 86 85 84", invert=True,
+            attrs=Attrs(io_standard="3.3-V LVTTL")),
+        
+        # Buttons, located on the bottom of the board, right of the LEDs.
+        *ButtonResources(
+            pins="88 89 90 91", invert=True,
+            attrs=Attrs(io_standard="3.3-V LVTTL")),
+
+        # Connections to the SKHynix RAM chip on board.
+        SDRAMResource(0,
+            clk="43", cs="72", we="69", ras="71", cas="70",
+            ba="73 74", a="76 77 80 83 68 67 66 65 64 60 75 59",
+            dq="28 30 31 32 33 34 38 39 54 53 52 51 50 49 46 44",
+            dqm="42 55", attrs=Attrs(io_standard="3.3-V LVCMOS")),
+
+        # Reset switch, located on the lower left of the board.
+        Resource("reset_switch", 0, PinsN("25", dir="i"), Attrs(io_standard="3.3-V LVTTL")),
```

Comment:
I would say this should be called `rst` and be assigned as `default_rst` in the class.

Reply:
Fixed.